tests/lwip_sock_ip/Makefile.ci

```diff
@@ -16,4 +16,9 @@ BOARD_INSUFFICIENT_MEMORY := \
     stm32f0discovery \
     stm32g0316-disco \
     stm32l0538-disco \
+    blackpill \
+    saml11-xpro \
+    saml10-xpro \
+    bluepill \
+    stm32mp157c-dk2 \
```

Comment:
```suggestion
    stm32mp157c-dk2 \
    nucleo-f302r8 \
```

And the same additions are also needed for `tests/lwip_sock_udp`. Note: Please keep this lists in alphabetical order. E.g. when using `vim` as editor you can visually select the lines to sort and use the `sort` command to do this.

Reply:
Both lists updated and sorted now